newsplease/crawler/simple_crawler.py

```diff
@@ -1,7 +1,74 @@
+# Based on https://github.com/adbar/trafilatura/blob/master/trafilatura/utils.py
```

Comment:
this falsely implies that the simple_crawler.py is based on utils.py from trafilatura, whereas simple_crawler.py was developed by the news-please team. please remove or move to a more specific part in this file.

Reply:
i think it will be better to move these first changes, esp. incl. the three functions you added to the top of simple_cralwer.py to a separate file. as simple_crawler.py is a core file of news please and to keep the architecture clean, it should only contain news-please related core functionality.

Reply:
Move related part to response_decoder.py